scripts/select-java

```diff
@@ -1,43 +1,55 @@
 #!/bin/bash
 
-has_java=false
+function select_java_debian() {
+    for version in "11" "8"; do
+        pkg_name=openjdk-${version}-jre-headless
+        if ! dpkg-query --status ${pkg_name}; then
+            continue
+        fi
+        java=$(dpkg -L ${pkg_name} | grep '/java$')
+        if [ -n "$java" ]; then
+            echo $java
+            return
+        fi
+    done
+}
+
+function select_java_fedora() {
+    for version in "11" "1.8.0"; do
+        pkg_name=java-${version}-openjdk-headless
+        if ! rpm --quiet -q ${pkg_name}; then
+            continue
+        fi
+        java=$(rpm -ql ${pkg_name} | grep '/java$')
+        if [ -n "$java" ]; then
+            echo $java
+            return
+        fi
+    done
```

Comment:
I think we can use [[ -e /usr/lib/jvm/jre-11/bin/java ]] and [[ -e /usr/lib/jvm/jre-8/bin/java ]], it appears a standard way. No need to mess with packages.

Reply:
Cool. Didn’t know this! Will update the fedora side.